web/src/app/schedules/ScheduleShiftList.js

```diff
@@ -266,17 +267,33 @@ export default class ScheduleShiftList extends React.PureComponent {
       <TextField
         fullWidth
         label='Time Limit (days)'
-        value={Duration.fromISO(this.props.duration).as('days')}
+        value={
+          this.state.isClear
+            ? ''
+            : Duration.fromISO(this.props.duration).as('days')
+        }
         disabled={this.props.activeOnly}
         max={30}
         min={1}
         type='number'
+        onBlur={() => {
+          this.setState({ isClear: false })
+        }}
         onChange={(e) => {
-          this.props.handleSetDuration(
-            Duration.fromObject({
-              days: clamp(1, 30, parseInt(e.target.value, 10)),
-            }).toISO(),
-          )
+          this.setState({ isClear: e.target.value === '' })
+          let val = e.target.value
+          if (parseInt(val, 10) > 30) {
+            val = '30'
+          } else if (parseInt(val, 10) < 1) {
+            val = '1'
+          }
```

Comment:
Doesn't look like this is used anymore

Reply:
It looks to be working for me — clamping the value at `30` if typing say `144` or at `1` if typing say `-3`

Reply:
Yes! beforehand, numerical values did not exceed 30 and that was handled by min & max props. This addition is not part of the acceptance criteria but it solves the scenario of what if a user specifies 60 or 90 days out, which is more than the 30 days. I wanted to mirror the former behavior by adding in the above.

Reply:
Clamping is already handled by the `clamp` function below this; `val` is unused, so this section of code can be dropped. 

`make check` would have flagged it as an unused variable but it is being read by `parseInt` in these conditionals and then re-assigned, after which it is never referenced again.

Reply:
Ah okay, I see what you mean now. This has been addressed and the changes have been made as suggested. Thanks for everyone's feedback!